tools/ingest.js

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env node
+
+/**
+ * How to use this script:
+ *
+ * Example: ingesting hca_mvp from dev to int-5
+ * npm run ingest hca_mvp -- --source-url=https://jade.datarepo-dev.broadinstitute.org --target-url=https://jade-5.datarepo-integration.broadinstitute.org --jc-path=~/broad/jade-data-repo-cli/./build/install/jadecli/bin/jadecli --target-env=broad-jade-int-5-data --target-name=hca_mvp --src-env=broad-jade-dev-data --bucket-name=hca_mvp
+ *
+ * make sure that you have the correct path to your jadecli install, if you have it aliased in your bashrc it should be the same path as there.
+ * also, make sure the source and target environments are the names of the DATA projects!!
+ *
+ * you may need to give yourself permissions to upload to the jade-testdata bucket as well
+ */
+
+const https = require('https');
+const _ = require('lodash');
+const fs = require('fs');
+const axios = require('axios');
+const { exec } = require('child_process');
+const inquirer = require('inquirer');
+const { argv } = require('yargs').command(
+  'ingest',
+  'copy a subset of production data into another environment',
+  (args) =>
+    args
+      .positional('datasetName', {
+        describe: 'name of the dataset to rolldown',
+        type: 'string',
+      })
+      .option('source-url', {
+        alias: 'sourceUrl',
+        desc: 'url for source environment',
+        default: 'broad-jade-dev-data',
+        type: 'string',
+      })
+      .option('target-url', {
+        alias: 'targetUrl',
+        desc: 'url for target environment',
+        default: 'broad-jade-dev-data',
+        type: 'string',
+      })
+      .option('target-env', {
+        alias: 'targetEnv',
+        desc: 'google (data) project for target environment',
+        default: 'dev',
+        type: 'string',
+      })
+      .option('src-env', {
+        alias: 'srcEnv',
+        desc: 'google (data) project for source environment',
+        default: 'dev',
+        type: 'string',
+      })
+      .option('target-name', {
+        alias: 'targetName',
+        desc: 'target dataset name',
+        default: 'datasetName',
+        type: 'string',
+      })
+      .option('jc-path', {
+        alias: 'jc',
+        desc: 'path to run jadecli',
+        default: '',
+        type: 'string',
+      })
+      .option('bucket-name', {
+        alias: 'bucketName',
+        desc: 'name of the bucket to place in jade-testdata',
+        default: '',
+        type: 'string',
+      })
+      .option('verbose', {
+        alias: 'v',
+        desc: 'print extra information',
+        default: false,
+        type: 'boolean',
+      }),
+);
+
+const CORE_BILLING_ACCOUNT_ID = '00708C-45D19D-27AAFA';
```

Comment:
Will this vary between environments? Also, say an HCA person wants to transfer data from dev to prod, who should pay for that transfer?